### PR TITLE
homepage&models-data

### PR DESCRIPTION
**Pull request by kishandobariy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ __pycache__/
 *.pyc
 
 db.sqlite3
+
+.vscode
```
Comment (sahildesai313) [inline .gitignore:8]: if your code is done after enter one space

File: myapp/admin.py [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
 
 class UserDetailAdmin(admin.ModelAdmin):
     list_display = ("username", "firstname", "lastname",
                     "phone", "email", "password", "confirmpassword")
+admin.site.register(UserDetail, UserDetailAdmin)
 
 
-admin.site.register(UserDetail, UserDetailAdmin)
+class Package_Details(admin.ModelAdmin):
+    list_display = ("package_name","no_of_day","package_description","package_price","image")
```
Comment (sahildesai313) [inline myapp/admin.py:18]: again, when code is done after enter one space

File: myapp/templates/master.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <title>{% block title %}{% endblock %}</title>
+</head>
+<body>
+
+{% block content %}
+{% endblock %}
+
+</body>
+</html>
```
Comment (sahildesai313) [inline myapp/templates/master.html:12]: again and again ,add space

File: myapp/templates/tour_detail.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      <li>{{ item.package_name }}</li>
+      <li>{{ item.no_of_day }}</li>
+      <li>{{ item.package_description }}</li>
+      <li>{{ item.package_price }}</li>
+      <li>{{item.image}}</li>
+      <a href="{{ item.photo.url }}">
+        <img src="{{ item.photo.url }}" alt="Lights" style="width:100%">
+   
+  </ul>
+  {% endblock %}
```
Comment (sahildesai313) [inline myapp/templates/tour_detail.html:15]: again ,respite problem

File: myapp/urls.py [hunk 5]
```diff
@@ -9,4 +11,6 @@
     path('profile/',views.profile,name="profile"),
     path('reg/',views.register,name='register'),
     path('home/',views.homepage,name='home'),
-]
\ No newline at end of file
+    path('home/tour_detail/',views.tour,name='tour'),
+   
+]+ static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (sahildesai313) [inline myapp/urls.py:16]: repeat problem

File: myapp/views.py [hunk 6]
```diff
@@ -111,6 +114,14 @@ def register(request):
...
-    user_data = UserDetail.objects.get(username=request.session["username"])
-    return render(request, 'home.html', context={"user": user_data})
+
+def tour(request):
+  item = package_details.objects.get()
+  template = loader.get_template('tour_detail.html')
+  context = {
+    'item': item,
+  }
+  return HttpResponse(template.render(context, request))
```
Comment (sahildesai313) [inline myapp/views.py:127]: repeat problem

**Review by sahildesai313 — CHANGES_REQUESTED** [review]

File: myapp/views.py (hunk 6 above)
Comment (kishandobariy) [reply myapp/views.py:127]: your review is done .....

File: myapp/models.py [hunk 7]
```diff
@@ -12,3 +12,19 @@ class UserDetail(models.Model):
...
+    package_description =models.TextField()
+    package_price=models.IntegerField()
+    image = models.ImageField(upload_to='pics')
+
+    def __str__(self):
+        return f"{self.package_name}{self.no_of_day}{self.package_description}{self.package_price}{self.image}"
+
+
+class Image(models.Model):
+    title = models.CharField(max_length=20)
```
Comment (sahildesai313) [inline myapp/models.py:29]: enter space

File: myapp/urls.py (hunk 5 above)
Comment (sahildesai313) [inline myapp/urls.py:16]: enter space

**Review by sahildesai313 — CHANGES_REQUESTED** [review]

File: myapp/models.py (hunk 7 above)
Comment (kishandobariy) [reply myapp/models.py:29]: enter space

File: myapp/templates/tour_detail.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+
```
Comment (sahildesai313) [inline myapp/templates/tour_detail.html:17]: space problem

**Review by sahildesai313 — CHANGES_REQUESTED** [review]

**Review by sahildesai313 — APPROVED** [review]

**Review by BhuvaRushika — APPROVED** [review]

File: student/urls.py [hunk 9]
```diff
@@ -17,7 +17,10 @@
...
 from django.urls import path,include
 
+
+
 urlpatterns = [
     path('admin/', admin.site.urls),
      path('',include('myapp.urls')),
-]
+
+]
```
Comment (khushali934) [inline student/urls.py:26]: enter space

**Pull request merged**